ui/app/models/config-ui/message.js

```diff
@@ -11,10 +11,22 @@ import { withFormFields } from 'vault/decorators/model-form-fields';
 const validations = {
   title: [{ type: 'presence', message: 'Title is required.' }],
   message: [{ type: 'presence', message: 'Message is required.' }],
+  link: [
+    {
+      validator(model) {
+        if (!model?.link) return true;
+        const [title] = Object.keys(model.link);
+        const [href] = Object.values(model.link);
+        if (!title && !href) return true;
```

Comment:
This line is unnecessary because if there are neither then `title || href` will be falsy which returns true 
<img width="433" alt="Screenshot 2024-01-19 at 1 39 22 PM" src="https://github.com/hashicorp/vault/assets/68122737/ae014925-5e80-459b-9ced-dc03c29644d8">
And you I think we should wrap `!!(title && href)` so it always returns a boolean

Reply:
haha that makes sense! Thanks for helping with this one 🙈